### PR TITLE
test: Compatible with int8 && int16, update test cases

### PR DESCRIPTION
**Pull request by aaravarr**
#### What type of PR is this?

test

#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.
- [ ] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

**Review by DMwangnima — COMMENTED** [review]
```EchoMultiByte``` and ```EchoMultiInt16``` in dubbo2kitex could work well for now? If these two cases work, please modify documents in **Type Mapping** part to tell users that they can pass List or Map of i8 and i16 types embedded in struct.

File: README.md [hunk 1]
```diff
@@ -43,7 +43,10 @@
 
 **重要提示**：
 1. 映射表中的 map 类型并没有被完全列举，当前仅包含经过测试的用例。
-   请勿在map类型中使用包含 **i8**、**i16** 和 **binary** 的键值。
+
+   请勿在 **map** 类型的 **返回值** 中使用包含 **i8**、**i16** 和 **binary** 的键值（dubbo-java -> dubbo-java 不支持），
```
Comment (felix021) [inline README.md]: `**map** 类型的 **返回值**` 是啥意思？

Comment (aaravarr) [reply README.md]: 就是thrift的service里面的接口响应字段，这里只是dubbo-java -> dubbo-java解码时不支持

Comment (felix021) [reply README.md]: 和 map 没啥关系？

Comment (felix021) [reply README.md]: 应该加个 3
> 3. 建议避开 dubbo-java 不兼容的用法，例如在接口的响应字段里使用 XXX

Comment (aaravarr) [reply README.md]: dubbo-java现在是不支持Map<Byte, Byte>和Map<Short, Short>的解码，但是支持这两个的编码。不过这两个我们都是支持的

Comment (aaravarr) [reply README.md]: 嗷好像 **map** 包含 **binary** 我们也不支持，给忽略了

File: tests/crosstest/go.mod [hunk 2]
```diff
@@ -25,7 +25,7 @@ require (
 	github.com/alibaba/sentinel-golang v1.0.4 // indirect
 	github.com/aliyun/alibaba-cloud-sdk-go v1.61.1704 // indirect
 	github.com/apache/dubbo-getty v1.4.9 // indirect
-	github.com/apache/dubbo-go-hessian2 v1.12.2 // indirect
+	github.com/apache/dubbo-go-hessian2 v1.12.3-0.20231101125246-fb070d002a57 // indirect
```
Comment (felix021) [inline tests/crosstest/go.mod]: v1.12.3 already released

**Review by felix021 — APPROVED** [review]

**Review by felix021 — APPROVED** [review]

**Pull request merged**